### ISSUE TITLE
[ubuntu touch] Canno't close the app from the quit button

### ISSUE THREAD

**Issue by TronFortyTwo** [loqui/im#1162]
If you press the button quit in the left panel, appear a banner on the top saying that LoquiM qill close soon, but then the banner disappear but the app doesn't close.

**Comment by nfsprodriver** [comment]
Problem seems to be here: https://github.com/loqui/im/blob/dev/src/scripts/loqui/menu.js#L410 Must be fit for Ubuntu Touch...

**Comment by nfsprodriver** [comment]
Is it possible to pass a system command into that function closing LoquiIM (app id)?

**Comment by avisoftware** [comment]
@nfsprodriver it is not the killall function
it is window.close()
https://github.com/loqui/im/blob/dev/src/scripts/loqui/menu.js#L413

**Comment by nfsprodriver** [comment]
Right, the killall says to logout from all account (App.js), right?

**Comment by avisoftware** [comment]
yes. my guess that the [req](https://github.com/loqui/im/blob/dev/src/scripts/loqui/menu.js#L405)  variable is undefined 
`var req = navigator.mozAlarms.getAll();`
so the timer has never set.

**Comment by avisoftware** [comment]
yes. my guess that req variable is undefined so the timer has never set.

**Comment by nfsprodriver** [comment]
So if it's not possible to make webcontainer call its closure, we should give a UI hint for that when button has been clicked.
